docs/dev/policy_documents/standards-for-tests.md

```diff
@@ -176,33 +178,25 @@ fi
 
 TEST_PKG_IDENT="${1}"
 export TEST_PKG_IDENT
+
 hab pkg install core/bats --binlink
 hab pkg install "${TEST_PKG_IDENT}"
-bats "$(dirname "${0}")/test.bats"
+bats "${TESTDIR}/test.bats"
 ```
 
 BATS test script `7zip/tests/test.bats`:
 
 ```bash
-TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
-@test "7z exe runs" {
```

Comment:
Why remove the `runs` tests?

Reply:
Added back.  I had initially removed them because of comments I'd received on other PRs, i.e., a version check on a binary more or less verifies the running of the binary.